scripts/release/update-sdk-ts-defs.js

```diff
@@ -4,6 +4,7 @@
  * to match the current definitions of functions of sdk.mjs.
  */
 
+import process from 'node:process';
 import fsSync from 'node:fs';
```

Comment:
The `fsSyns.exists` can be replaced with the Promise function `fs.access`.

```diff
-import fsSync from 'node:fs';
+import fs from 'node:fs/promises';

-const fileExists = fsSync.existsSync(sdkMts);
+const fileExists = await fs.access(sdkMts).then(() => true).catch(() => false);
```

Reference: https://nodejs.org/api/fs.html#fspromisesaccesspath-mode